implementations/python/python/ockam/logging/logging.py

```diff
@@ -141,3 +144,31 @@ def debug(msg, *args, **kwargs):
 def error(msg, *args, **kwargs):
     logger = logging.getLogger("node")
     logger.error(msg, stacklevel=2, *args, **kwargs)
+
+
+class LoggerAware(Protocol):
+    logger: logging.Logger
+
+
+class InfoContext(LoggerAware):
+    @contextmanager
+    def info(self, before_msg, after_msg):
+        self.logger.info(before_msg)
+        try:
+            yield
+        except Exception as e:
+            self.logger.info(f"{e}")
+            return
+        self.logger.info(after_msg)
+
+
+class DebugContext(LoggerAware):
+    @contextmanager
+    def info(self, before_msg, after_msg):
```

Comment:
Why is this function called `info`? I'd expect it to be `debug`

Reply:
Thanks, this is fixed now